docs/README.md

```diff
@@ -34,7 +34,8 @@ helm delete codacy
 
 
 ## Requirements
-[All requirements](requirements/index.md)
+- [All requirements](requirements/index.md)
```

Comment:
Do you need to split these documents?

Reply:
I think we should start simple and putting everything in a single doc and split if necessary whenever is needed

Reply:
I felt that an index file would just be a listing of all the documents in its directory. Therefore the name index.